core/lexicon/en/default.inc.php

```diff
@@ -52,6 +52,7 @@
 $_lang['categories'] = 'Categories';
 $_lang['category'] = 'Category';
 $_lang['category_create'] = 'Create Category';
+$_lang['category_create_desc'] = 'A category without elements is displayed only in the list of all Categories.';
```

Comment:
In my opinion, you complicate the text (you actually described the definition of the category) :)
For user it is clear what a category is and how they work, because the user has ALREADY called the category creation form.
The problem is that the new category appears ONLY in the list of Categories when it is created, which is exactly what I was trying to make clear.

Reply:
Here is a slightly more detailed version:

`A category for elements will only be displayed if at least one element is assigned to it (by default, all categories are displayed in the Categories list).`

Reply:
Yes, I thought about the fact that I was more fully defining what a Category is as well, which I elected to keep in there only because these pop up windows are the only place you can define them (as opposed to the other elements that have their own panels where each is defined at the top). Re the second sentence in my edit, although  I come at it from a different angle, I thinks illustrates your point (just in a different way). If it still doesn't seem right to you, what about:

"Note that, until they have been assigned to one or more Elements, you will only see the Categories you create listed in the Categories folder."